src/encoded/schemas/file.json

```diff
@@ -364,11 +375,19 @@
                         }
                     }
                 },
+                {
+                    "required": ["index_of"],
+                    "properties": {
+                        "output_type": {
+                            "enum": ["index reads"]
+                        }
+                    }
+                },
                 {
                     "not": {
                         "properties": {
                             "output_type": {
-                                "enum": ["reads", "restriction enzyme site locations"]
+                                "enum": ["reads", "index reads", "restriction enzyme site locations"]
```

Comment:
In general, I consider "index reads" as similar to "reads". So at line 397 below here for "assembly" dependency. Should "index read" be not able to have assembly?

Reply:
you are right, fixed